doc/source/point-in-time-recovery.rst

```diff
@@ -0,0 +1,84 @@
+
+|PITR| is available in |PBM| starting from v1.3.0. |PITR| is enabled via the ``pitr.enabled`` config option.
+
+.. code-block:: bash
+
+   $ pbm config --set pitr.enabled=true
+
+Incremental backups
+================================================================================
+
+When |PITR| is enabled, |pbm-agent| starts slicing the :term:`oplog <Oplog>`. The |pbm-agent| to do the slicing is elected in the same way as secondary nodes elect a new primary. (Find more information in :ref:`pbm.architecture.agent`.)
+
+A slice covers a 10 minute span of oplog events. It can be shorter if |PITR| is disabled or if slicing is stopped by a backup operation.
+
+The oplog slices are stored in the :file:`pbmPitr` subdirectory in the :ref:`remote storage defined in the config <pbm.config.initialize>`. A slice name reflects the start and end time this slice covers. 
+
+The |pbm-list| output includes both backup snapshots and valid time ranges for recovery. It also shows the |PITR| status.
+
+.. code-block:: bash
+
+   $ pbm list
+
+   Backup snapshots:
+  	2020-07-10T12:19:10Z
+  	2020-07-14T10:44:44Z
+  	2020-07-14T14:26:20Z
+  	2020-07-17T16:46:59Z
+   PITR <on>:
+  	2020-07-14T14:26:40 - 2020-07-16T17:27:26
+  	2020-07-17T16:47:20 - 2020-07-17T16:57:55
+
+.. note::
+
+   If you just enabled |PITR|, the time range list in the ``pbm list`` output is empty. It requires 10 minutes for the first chunk to appear in the list.  
+
+Restore to the point in time
+============================
+
+.. important::
+
+   Since a restore operation breaks the time line of oplog slicing, you must disable |PITR| before restoring a database:
+
+   .. code-block:: bash
+   
+      $ pbm config --set=pitr.enabled=false
+
+Run |pbm-restore| and specify the time stamp from the valid range:
+
+.. code-block:: bash
+
+   $ pbm restore --time="2020-07-14T14:27:04"
+
+.. seealso::
+
+   :ref:`pbm.running.backup.restoring`.
+
+All oplog slices made after the restore time stamp and before the last backup become invalid. Therefore, after the restore is complete, make a new backup to serve as the starting point for oplog slicing: 
+
+.. code-block:: bash
+
+   $ pbm backup
+
+Re-enable |PITR| to resume oplog slicing:  
+
+.. code-block:: bash
+
+   $ pbm config --set=pitr.enabled=true
+
+
+.. rubric:: Delete a backup
+
+When you :ref:`delete a backup <pbm.backup.delete>`, all oplog slices that are made after the specified backup or that are older than the specified time will be deleted.  
+
+.. note::
+
+   When |PITR| is enabled, the most recent backup snapshot won't be deleted.
```

Comment:
Just to be clear. All PITR chunks being deleted only with the snapshot they are related to. So if one deletes the snapshot then and only then any chunks related to this snapshot should be deleted. There is no other way to delete a chunk (at least for the time being)

Ramifications of that:
> all oplog slices that are made after the specified backup or that are older than the specified time will be deleted  

if you specify a particular snapshot, PITR chunks that are related to the next snapshot (if such snapshot exists) won't be deleted:
```
---bcp1----bcp2---bcp3---
$ pbm delete-backup bcp2
---bcp1----bcp3---
```
\* `bcp<N>` - is a snapshot, `-` - oplog slice

again, if we specify a time - some chunks that a younger than that time but related to the deleted snapshots would be deleted as well
```
---bcp1----bcp2------bcp3---
                 ^ - t1
$ pbm delete-backup --older-than t1
---bcp1----bcp3---
```
\* all chunks after bcp2 and up to bcp3 would be deleted

> When |PITR| is enabled, the most recent backup snapshot won't be deleted.

Any chunks related to that backup won't be deleted either

Reply:
> There is no other way to delete a chunk (at least for the time being)

Well, you can delete the files in the pitrChunks/ subdirectory in the remote storage yourself.
I make a point of mentioning this because a user might turn on PITR, discover that it takes up a huge amount of space because write churn is large, disable it, and want to just keep snapshots.

Reply:
@akira-kurogane yes good point. But I'd recommend running `pbm config --force-resync` after deleting files on the remote storage to avoid the misleading output of `pbm list`

Reply:
Got it.

Anyhow this is comments on comments, I don't suggest an edit to the *.rst file content above.